Tests/ApolloCodegenTests/AnimalKingdomAPI/ExpectedGeneratedOutput/Queries/AllAnimalsQuery.swift

```diff
@@ -276,7 +297,7 @@ public struct AllAnimalsQuery: GraphQLQuery {
 
         public var isJellicle: Bool { data["isJellicle"] }
         public var bodyTemperature: Int { data["bodyTemperature"] }
-        public var height: Height { data["height"] }
+        public var height: AllAnimal.AsPet.AsWarmBlooded.Height { data["height"] }
```

Comment:
Is this the intended change? I thought after we talked yesterday we decided not to go this route?